.github/workflows/C++.yml

```diff
@@ -0,0 +1,79 @@
+name: C++ Build
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  Linux:
+    runs-on: ubuntu-latest
+    container: quay.io/pypa/manylinux2014_x86_64:latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install fftw
+      run: |
+        yum install -y fftw3-devel
+
+    - name: Compile C++ code
+      run: |
+        make lib
```

Comment:
looks good. keep in mind spreadtestall is lower-level than lib. So could swap these and it'll test the spreader first.

Reply:
will swap the order